backend/app/api/v1/file.go

```diff
@@ -803,7 +781,6 @@ func (b *BaseApi) ReadFileByLine(c *gin.Context) {
 
 // @Tags File
 // @Summary Batch change file mode and owner
-// @Description 批量修改文件权限和用户/组
 // @Accept json
 // @Param request body request.FileRoleReq true "request"
 // @Success 200
```

Comment:
As of October 2021, it is no longer recommended to implement `@Accept` and related tags at top level due to its deprecated usage. I would recommend refactoring these imports according to the current context in 2024.

For instance:

```go
import (
	"github.com/gin-gonic/gin" // use actual import path here

// Some other packages ...
```

Please let me know if you require specific recommendations regarding this particular example.